### ISSUE TITLE
lost connection device. Exited (sigterm)

### ISSUE THREAD

**Issue by siamandkm** [microsoft/vscode#130360]
I work with vscode. And I am connected to the phone via adb wifi, but sometimes it shows a lost connection to device error. I have to restart the program from the beginning. Or sometimes my phone restarts. If you can help me, thank you.

**Comment by rzhao271** [comment]
Did this problem start recently?
Are there any steps to reproduce the issue, or does it happen randomly?

**Comment by siamandkm** [comment]
Yes, it started recently. I even reinstalled Windows and programs

**Comment by rzhao271** [comment]
Does downgrading to an older version of VS Code resolve the issue?
You can find older versions by clicking the month and then looking through the download links below the release notes header at https://code.visualstudio.com/updates/v1_59.
I also wonder whether the problem could be caused by an extension.

**Comment by siamandkm** [comment]
> Does downgrading to an older version of VS Code resolve the issue?
> You can find older versions by clicking the month and then looking through the download links below the release notes header at https://code.visualstudio.com/updates/v1_59.
> I also wonder whether the problem could be caused by an extension.

I have the same problem in Android Studio, I also updated the paltform tools

**Comment by siamandkm** [comment]
My problem is very strange. I will be involved for a month
